examples/systemd/sysconfig.node_exporter

```diff
@@ -0,0 +1 @@
+OPTIONS="-collector.textfile.directory /var/lib/node_exporter/textfile"
```

Comment:
/var/lib/node_exporter/textfile_collector is what my debs use, @TheTincho's Debian debs use a different path. No matter what you put in here, it's going to break for someone.